src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceDetails.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <v-row>
+    <v-col cols="12" md="12" lg="12" xl="12">
+      <h3>Work experience hours</h3>
+      <div role="doc-subtitle">Please provide as much detail as possible.</div>
+      <v-row class="mt-5">
+        <v-col cols="12" md="8" lg="6" xl="4">
+          Applicant indicated you observed them work:
+          <b>50 hours</b>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            type="number"
+            :rules="[Rules.required('Enter a whole number greater than zero (0)')]"
+            label="Total number of hours you observed the applicant working"
+            variant="outlined"
+            color="primary"
+            maxlength="10"
+            hide-details="auto"
+            @input="updateField('hours', $event)"
+          />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <p>How often did the applicant work or valunteer?</p>
+          <v-radio-group hide-details="auto" :rules="[Rules.requiredRadio('Select an option')]" @update:model-value="updateField('workHoursType', $event)">
+            <v-radio v-for="(workHoursType, index) in workHoursTypeRadio" :key="index" :label="workHoursType.label" :value="workHoursType.value"></v-radio>
+          </v-radio-group>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <h3>Program and age of children</h3>
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            :rules="[Rules.required('Enter the name of child care program')]"
+            label="Name of child care program"
+            variant="outlined"
+            color="primary"
+            maxlength="100"
+            hide-details="auto"
+            @input="updateField('childrenProgramName', $event)"
+          />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-autocomplete
+            label="Type of child care program"
+            variant="outlined"
+            color="primary"
+            :rules="[Rules.required('Select an option')]"
+            :items="childrenProgramTypeDropdown"
+            hide-details="auto"
+            @update:model-value="updateField('childrenProgramType', $event)"
+          ></v-autocomplete>
```

Comment:
You probably need some custom logic here. That if the program type isn't Other. We also clear up the data in the wizard. 

This handles the case where someone clicks Other. Inputs info then switches off. 
Our wizard would still have info for the Other in that case. 

You can follow what was done in the characterReferenceEvaluation for Relationship Type other.